src/main/java/uk/gov/justice/probation/courtcasematcher/security/OAuth2SecurityConfig.java

```diff
@@ -26,7 +26,7 @@ public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
             .and().oauth2Client()
             .and()
             .authorizeHttpRequests(authorize -> authorize
-                .requestMatchers("/actuator/health", "/actuator/health/ping", "/queue-admin/retry-all-dlqs", "/replay404Hearings").permitAll()
+                .requestMatchers("/actuator/health/**", "/actuator/health", "/queue-admin/retry-all-dlqs", "/replay404Hearings").permitAll()
```

Comment:
this isn't the same path which the tests are verifying - it begins with `/actuator`. Does this change make the tests pass?

Reply:
hi, yep the test has /actuator defined in it near the top of the file

```
@BeforeEach
    public void before() {
        TestConfig.configureRestAssuredForIntTest(port);
        RestAssured.basePath = "/actuator";
    }
```